lib/node_modules/@stdlib/repl/cli/lib/commands/example.js

```diff
@@ -162,7 +162,7 @@ function command( repl, cmds ) {
 					lines.push( out[ i ] );
 
 					// If line is part of a multi-line input, wait for the next line...
-					if ( repl._multilineHandler.isMultilineInput( lines.join( '\n' ) ) ) {
+					if ( repl._isMultilineInput( lines.join( '\n' ) ) ) {
```

Comment:
You're calling a private method of the private. Don't do this. Make it a public method.